pkg/projects/functions.go

```diff
@@ -26,8 +28,49 @@ func (tc *TestContext) CheckMessageInAppLog(edgeNode *device.Ctx, appID uuid.UUI
 			log.Fatalf("LogAppsChecker: %s", err)
 		}
 		if foundedMessage != "" {
+			for _, clb := range callbacks {
+				clb()
+			}
 			return fmt.Errorf("founded in app logs: %s", foundedMessage)
 		}
 		return nil
 	}
 }
+
+//SendCommandSSH try to access SSH with timer and sends command
+func SendCommandSSH(ip *string, port *int, user, password, command string, foreground bool, callbacks ...Callback) ProcTimerFunc {
```

Comment:
I will merge that but please review all ssh functions in eden and leave as  few as possible )